### DISCUSSION TITLE
Numerical solutions (read: code) as answer supplements?

### DISCUSSION THREAD

**Discussion by Per48edjes** [fifthist/Introduction-To-Probability-Blitzstein-Solutions#24] [Ideas]
To that end, when I was working through the STAT 110 course, I wrote [a handful of notebooks](https://github.com/Per48edjes/Harvard-STAT110/) to help build intuition, solve problems numerically, simulate processes, etc.  for certain problems.

Would contributing along these lines be helpful? Or out of scope? (Either way, would love to pitch in where I can!)

**Comment by fifthist** [comment]
Hello Ravi, are we talking about  Jupyter notebooks? If so, I think these
would be a fantastic resource! I can create a "notebooks" directory to host
them. Any ideas how you'd name them to relate to particular
problems/sections?

**Reply by Per48edjes** [reply]
> Hello Ravi, are we talking about  Jupyter notebooks? If so, I think these
would be a fantastic resource!

Great! (See the linked Jupyter notebooks above for examples of what I'm talking about.)

> Any ideas how you'd name them to relate to particular
problems/sections?

The ones I've done are from the Strategic Practice (problem sets) from [the course](https://projects.iq.harvard.edu/stat110/strategic-practice-problems), but I believe at least some of these problems draw from the book. Happy to adopt whatever naming convention you have in mind (e.g., mimicking what you've done in `src/`).
